theme/js/dynamic-scripts.js

```diff
@@ -0,0 +1,65 @@
+$(document).ready(function() {
+    getBlogCardComponentData();
+
+    appendCountryData();
+
+    getBlogContentData();
+});
+
+function getBlogCardComponentData() {
+    $.getJSON('theme/json/posts.json', function(data) {
+        // Run function for each card. The post variable here represents the json dataset.
+        $.each(data, function(i, post) {
+            $('.card-component-container--horizontal .card-wrapper')
+            // Insert the following markup and json data into the card-component-container div.
+                .append('<div class="card-wrapper"><div class="card card-one clickable card--horizontal"><div class="card-image-wrapper"><img src="'+post.imageLink+'" alt="'+post.imageAltText+'" class="card-image responsive-image"></div><div class="card-text-container"><h3>'+post.title+'</h3><p>'+post.datePosted+'</p><p>'+post.summary+'</p><a href="'+post.link+'?postId='+post.id+'">'+post.linkText+'</a></div></div></div>');
+        });
+    });
+    setTimeout(function(){
+        clickableElement($('.card-component-container .row-container--horizontal'));
+    }, 100);
+}
+
+function appendCountryData() {
+    // Run function only on the help page.
+    if (window.location.pathname === '/help.php') {
+        // Link to json data.
+        $.getJSON('http://vocab.nic.in/rest.php/country/json', {
+            format: "json"
+            // Once the json has been loaded, trigger the following function.
+        }).done(function (data) {
+            /* Run function for each country in the json data.
+            countryData is a variable representing the json dataset at the countries array level.*/
+            $.each(data['countries'], function (i, countryData) {
+
+                // Insert markup and json data into the select element.
+                $('.select-country')
+                    .append('<option value="' + countryData.country.country_id + '">' + countryData.country.country_name + '</option>');
+            });
+            // If the json data cannot be loaded, display an error message.
+        }).fail(function (data, textStatus, error) {
+            var err = textStatus + ", " + error;
+            console.log("Request Failed: " + err);
+        });
+    }
+}
+
+function getBlogContentData() {
+    // Run function only on the blog-content page.
+    if (window.location.pathname === '/blog/blog-content.php') {
+        // Setting the variable postId to equal the postId taken from the URL.
+        var postId = $.getUrlParameter('postId');
+        // If the postId variable is empty, set it to 0.
+        if (postId === null) {
+            postId = '0';
```

Comment:
Integers ( numbers ) don’t need to have quotes around them, this is only for strings.

Reply:
okay, done.